packages/focusvisible/stories.tsx

```diff
@@ -22,7 +26,7 @@ const StyledCustomFocus = styled.div`
   &[data-garden-focus-visible] {
     outline: 2px dashed red;
   }
-`;
+` as React.FC<IStyledCustomFocus>;
```

Comment:
Let's take some time and pair on the weirdness that is styled-components and TypeScript. There are some unique ways to type this. Using the following should work:

```tsx
const StyledCustomFocus = styled.div<IStyledCustomFocus>``;
```

Reply:
Oh yes, that works. The styled component is no longer casted. Thanks!